capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name
+  end
+end
```

Comment:
- [ ] Please know that the function `correct_name` should return the name capitalized like this: `@nameable.correct_name.capitalize`